src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,20 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+export const booksSlice = createSlice({
+  name: 'books',
+  initialState: {
+    value: [],
+  },
+  reducers: {
+    add: (state, action) => {
+      const { title } = action.payload;
+      state.value.push({ title });
+    },
```

Comment:
## Recommendation Changes ♻️

- [ ] Good job with the state title implementation for a book, however, we're expected to have the author as well. It's a slight change and I trust you will add the author store value next to the title. Hence, I recommend you work on that and decide not to ask for a required change 👍🏽